src/Form/Constraint/YoutubeValidator.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Message\Cog\Form\Constraint;
+
+use Symfony\Component\Validator;
+
+/**
+ * Class YoutubeValidator
+ * @package Message\Cog\Form\Constraint
+ *
+ * @author  Thomas Marchant <thomas@mothership.ec>
+ */
+class YoutubeValidator extends Validator\Constraints\UrlValidator
+{
+	const URL_MATCH         = 'youtu';
+	const CODE_LENGTH       = 11;
+	const ERROR_MESSAGE     = '\'%value%\' is not a valid Vimeo URL';
+	const VALUE_KEY         = '%value%';
+
+	/**
+	 * {@inheritDoc}
+	 */
+	public function validate($value, Validator\Constraint $constraint)
+	{
+		if (null === $value || false === $value || '' === $value) {
```

Comment:
Might be worth a comment on what this does. What should I expect returning early from this method to do?